### PR TITLE
trdsql: add page

### PR DESCRIPTION
**Pull request by vanvuvuong**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by vanvuvuong** [comment]
Updated all the suggestions

File: pages/common/trdsql.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# trdsql
+
+> CLI tool that executes SQL on CSV, LTSV, JSON, YAML, TBLN files.
```
Comment (spageektti) [inline pages/common/trdsql.md:3]:
```suggestion
> Execute SQL on CSV, LTSV, JSON, YAML, and TBLN files.
```
Style guide.

**Comment by spageektti** [comment]
Can you please convert all examples to be in imperative mood? See style guide

**Comment by vanvuvuong** [comment]
I just updated, please review

File: pages/common/trdsql.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convert object data from multiple JSON files to a CSV file with header (`-oh`) & double quote:
+
+`trdsql -ocsv -oh "SELECT * FROM {{path/to/file/*.json}}" | sed 's/\([^,]*\)/"&"/g' > {{path/to/file.csv}}`
+
+- Interpret JSON list as a table & put object inside as columns (path/to/file.json: `{"list":[{"age":"26","name":"Tanaka"}]}`):
+
+`trdsql "SELECT * FROM {{path/to/file.json}}::.list`
+
+- Complex SQL data manipulation with multiple CSV files with first line is header (`-ih`):
```
Comment (spageektti) [inline pages/common/trdsql.md:14]:
```suggestion
- Manipulate Complex SQL data with multiple CSV files with first line is header (`-ih`):
```

File: pages/common/trdsql.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`trdsql -icsv -ih "SELECT {{column1,column2}} FROM {{path/to/file*.csv}} WHERE column2 != '' ORDER BY column1 GROUP BY column1"`
+
+- Merge content of 2 CSV files to one CSV file:
+
+`trdsql "SELECT {{column1,colum2}} FROM {{path/to/file1.csv}} UNION SELECT {{column1,column2}} FROM {{path/to/file2.csv}}"`
+
+- Simple connection to PostgreSQL database:
```
Comment (spageektti) [inline pages/common/trdsql.md:22]:
```suggestion
- Connect to PostgreSQL database:
```

File: pages/common/trdsql.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`trdsql -driver postgres -dsn "host=localhost port=5433 dbname=trdsql_test" "SELECT 1"`
+
+- Create table data to MySQL database from CSV file:
+
+`trdsql -driver mysql -dsn "noborus:noborus@/trdsql_test" -ih "CREATE TABLE {{table}} ({{column1}} int, {{colum2}} varchar(20)) AS SELECT {{column3}} AS {{column1}},{{column2}} FROM {{path/to/header_file.csv}}"`
+
+- Simple showing data from compress log files:
```
Comment (spageektti) [inline pages/common/trdsql.md:30]:
```suggestion
- Show data from compress log files:
```

File: pages/common/trdsql.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# trdsql
+
+> Execute SQL on CSV, LTSV, JSON, YAML, and TBLN files.
+> More information: <https://noborus.github.io/trdsql/>.
+
+- Convert object data from multiple JSON files to a CSV file with header (`-oh`) & double quote:
```
Comment (gutjuri) [inline pages/common/trdsql.md:6]:
```suggestion
- Convert object data from multiple JSON files to a CSV file with header (`-oh`) and double quote:
```

File: pages/common/trdsql.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`trdsql -ocsv -oh "SELECT * FROM {{path/to/file/*.json}}" | sed 's/\([^,]*\)/"&"/g' > {{path/to/file.csv}}`
+
+- Interpret JSON list as a table & put object inside as columns (path/to/file.json: `{"list":[{"age":"26","name":"Tanaka"}]}`):
```
Comment (gutjuri) [inline pages/common/trdsql.md:10]:
```suggestion
- Interpret JSON list as a table and put objects inside as columns (path/to/file.json: `{"list":[{"age":"26","name":"Tanaka"}]}`):
```

**Review by gutjuri — COMMENTED** [review]
Thanks, two sggestions

**Comment by vanvuvuong** [comment]
Thank you for your comments, I just fixed, please tag me if there are somethings that need to change.

**Review by spageektti — APPROVED** [review]

**Pull request merged**